tests/net/net_pkt/src/main.c

```diff
@@ -259,7 +259,11 @@ static void test_net_pkt_basics_of_rw(void)
 	/* And re-read the value again */
 	net_pkt_cursor_init(pkt);
 	net_pkt_skip(pkt, 1);
-	net_pkt_read_be16(pkt, &value16);
+	if (net_pkt_read_be16(pkt, &value16) != 0) {
```

Comment:
Just use proper zassert variant here instead of this check + printk.

Reply:
Maybe I'm not clear understand you, but please check my changes again.

Reply:
@maxxlife, using zassert_equal() would be even better. Just see the line below for example usage.